api/src/controllers/cohort-session.js

```diff
@@ -81,4 +83,93 @@ router.get("/availability/2022", passport.authenticate("young", { session: false
   }
 });
 
+router.post("/eligibility/2022", async (req, res) => {
+  const eligibilityObject = {
+    department: req.body.department,
+    birthDate: req.body.birthDate,
+    schoolLevel: req.body.schoolLevel,
+  };
+  const { error, value } = Joi.object({
+    department: Joi.string().required(),
+    birthDate: Joi.string().required(),
+    schoolLevel: Joi.string(),
```

Comment:
si on ne cochait pas la case, ca plantait (sans message d'erreur d'ailleurs)
```suggestion
    schoolLevel: Joi.string().allow(null, ""),
```